sse2neon.h

```diff
@@ -4884,6 +4884,21 @@ FORCE_INLINE __m128i _mm_cvtps_epi32(__m128 a)
 #endif
 }
 
+// Converts the four single-precision, floating-point values of a to signed
+// 16-bit integer values.
+//
+//   r0 := (short) a0
+//   r1 := (short) a1
+//   r2 := (short) a2
+//   r3 := (short) a3
+//
+// https://software.intel.com/sites/landingpage/IntrinsicsGuide/#text=_mm_cvtps_pi16
+FORCE_INLINE __m64 _mm_cvtps_pi16(__m128 a)
+{
+    return vreinterpret_m64_s16(
```

Comment:
For Aarch64, would it be better to use `vmovn_s32(vcvtq_s32_f32(vrndiq_f32(a)))`?

Reply:
In my lastest commit (868a0472abb7cf15eb85b69cbca858a6bd6ea4f2), I have improved things and I use _mm_cvtps_epi32 as the base for float to integer conversion. This function uses round to nearest directly (no need for 2 instructions).